<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dmax_output_Dialog</class>
 <widget class="QDialog" name="dmax_output_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>109</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dmax_output_dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="disabled_radioButton">
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="text">
      <string>Disabled</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="ArtNet_radioButton">
     <property name="text">
      <string>Art-Net</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="sACN_radioButton">
     <property name="text">
      <string>sACN</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
